esx_service/vmdk_ops.py

```diff
@@ -847,16 +848,31 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
     if not datastore:
         datastore_url = default_datastore_url
         datastore = default_datastore
+        use_default_ds = True
```

Comment:
At line 846 shouldn't the "known datastores" be the ones assigned to the vmgroup - the default and any more?